dashboard/src/features/accessControl/constants.js

```diff
@@ -20,3 +20,15 @@ export const policyOptions = [
     hint: 'Access to monitoring-specific endpoints'
   },
 ]
+
+export const subjectFieldOptions = [
```

Comment:
Note for posterity: Node SDK has been updated so it doesn't snakeize all-caps strings.